### ISSUE TITLE
Merging this project in API Platform

### ISSUE THREAD

**Issue by dunglas** [nilportugues/symfony-jsonapi#11]
Hi,

I just find your project and it's very interesting. In our (not stable yet) API Platform v2, we have decoupled the JSON-LD/Hydra part from the engine. Adding support for new formats (while still having all other features) is just a matter of creating a normalizer for it (what you've done in this bundle): https://github.com/api-platform/doc/blob/master/api-bundle/content-negotiation.md

I'm working on HAL support, but having JSON API support as well as other hypermedia format in a same project would be a big win both for API Platform and the Symfony ecosystem.

Are you interested for collaborating on this topic?

**Comment by nilportugues** [comment]
Yeah sure. I've got a HAL transformer done too. The serializer it uses is my own, but I remember all the logic to port it somewhere. 

I'm certain we can get to talk and find a way to merge projects,as your v2 is basically what I was aiming too do too. 

So yeah! :+1:
